addons/web/static/src/core/errors/error_dialogs.xml

```diff
@@ -54,7 +54,11 @@
       <Dialog title="this.constructor.title" size="'md'" contentClass="'o_error_dialog'">
          <div role="alert">
           <p class="text-prewrap">
-            <p><b>An error occurred</b></p>
+            <p><b>
+                An error occurred
+                <br/>
+                <code t-if="props.serverTime">Server time: <t t-esc="props.serverTime"/></code>
```

Comment:
Not sure about the design. Maybe we want something more discreet.

Reply:
actually, I was thinking the opposite. Some client have the bad habitude to take blurry pictures with their phone of the screen. If at least the time is readable, we can have a good starting point for the investigation. But I find <b><code> not very beautiful to read